docs/source/commands/supergraphs.mdx

```diff
@@ -219,6 +164,73 @@ In some cases, you might want Rover to skip updating its composition library to
 
 In these cases, you can pass the `--skip-update` flag to `rover supergraph compose`. If you provide this flag, your `supergraph.yaml` file must specify a `federation_version` (which is recommended regardless).
 
+### Composition output format
```

Comment:
These sections are untouched, just moved from above. (Disappointed in you GH diffing)